manager/allocator/networkallocator/networkallocator.go

```diff
@@ -282,24 +327,32 @@ func (na *NetworkAllocator) IsTaskAllocated(t *api.Task) bool {
 	}
 
 	// To determine whether the task has its resources allocated,
-	// we just need to look at one network(in case of
+	// we just need to look at one global scope network (in case of
```

Comment:
Correct. If the task is _only_ connected to node-local networks, from Allocator's perspective it has to be considered allocated, no allocation needs to be done for it.
If the task is not connected to any networks, then it is to be considered not allocated, as today.